java/ql/src/experimental/Security/BigDecimalDOS/BigDecimalDOS.ql

```diff
@@ -0,0 +1,38 @@
+/**
+ * @id java/BigDecimalDOS
+ * @name Java-BigDecimal-DOS-Vulnerability
```

Comment:
The name does not have to use hyphens, see [query metadata style guide](https://github.com/github/codeql/blob/main/docs/query-metadata-style-guide.md#query-name-name)